cde-core/resource/resources/custom/components/cgg/cgg-implementation.js

```diff
@@ -20,6 +20,10 @@ var CggComponent = UnmanagedComponent.extend({
   update: function() {
     this.detectSvg();
     this.clear();
+    // get actual parameters
```

Comment:
Don't forget the (AMD version)[https://github.com/webdetails/cde/blob/master/cde-core/resource/resources/custom/amd-components/cgg/CggComponent.js#L42] and to add a (test)[https://github.com/webdetails/cde/blob/master/cde-core/test-js/components/CggComponent-spec.js]. The legacy version doesn't have tests, only the AMD.